python/sglang/srt/server_args.py

```diff
@@ -20,6 +20,8 @@
 import tempfile
 from typing import List, Optional
 
+from vllm.transformers_utils.utils import check_gguf_file
```

Comment:
copy these small and stable functions into `hf_transformer_utils.py` instead of importing

Reply:
Done